### PR TITLE
HTTP query queue speedup

### PR DESCRIPTION
**Pull request by AdityaHegde**

### CODE REVIEW

**Comment by nishantmonu51** [comment]
1. Should we also increase the default queue size to 20 ? 
2. would it also be possible to adapt the default to 200 if the app is hosted over https or on a remote server?

**Comment by AdityaHegde** [comment]
Updated queue size to 20. Also if it is https with a non localhost hostname then it is set to 200.

**Comment by hamilton** [comment]
@magorlick would you mind taking a look at how this feels? This PR will hopefully make modeling feel a bit smoother / faster.

File: web-local/src/lib/http-request-queue/HttpRequestQueue.ts [hunk 1]
```diff
@@ -15,21 +15,29 @@ import {
...
 export const UrlExtractorRegex =
   /v1\/instances\/[\w-]*\/(metrics-views|queries)\/([\w-]*)\/([\w-]*)\/(?:([\w-]*)(?:\/|$))?/;
 
 // intentionally 1 less than max to allow for non profiling query calls
-const QueryQueueSize = 5;
+let QueryQueueSize = 20;
+try {
+  if (
+    window.location.protocol === "https:" &&
+    window.location.host !== "localhost"
```
Comment (nishantmonu51) [inline web-local/src/lib/http-request-queue/HttpRequestQueue.ts:27]: should this be https OR localhost instead of AND ?

**Comment by hamilton** [comment]
@AdityaHegde Here's what I'm seeing in this branch:
- it appears the cardinality + null plots get batch updated in one pass; that feels a little funky to me. Ideally we get that information ASAP and then get the ordering right.
- it also appears that the cardinality + null plots are enqueued after the numeric histograms. They should probably be queried first
- I'm not getting any kind of error feedback in the preview table when I type, no errors of any kind

https://user-images.githubusercontent.com/95735/209019795-6aefcff6-6aea-469e-808a-97622a0aca6c.mp4

**Comment by AdityaHegde** [comment]
@hamilton This looks like an issue on main itself. @magorlick also faced similar issues with errors.

**Comment by AdityaHegde** [comment]
@hamilton Fixed them. Please check now.

File: web-local/src/lib/http-request-queue/HttpRequestQueue.ts (hunk 1 above)
Comment (AdityaHegde) [reply web-local/src/lib/http-request-queue/HttpRequestQueue.ts:27]: Updated

**Review by nishantmonu51 — APPROVED** [review]
👍

**Pull request merged**